test/v2/connectors/DeleteHistoricUkPropertyAnnualSubmissionConnectorSpec.scala

```diff
@@ -20,43 +20,65 @@ import api.connectors.{ConnectorSpec, DownstreamOutcome}
 import play.api.libs.json.JsObject
 import api.models.domain.{HistoricPropertyType, Nino, TaxYear}
 import api.models.outcomes.ResponseWrapper
+import mocks.MockFeatureSwitches
 import v2.models.request.deleteHistoricUkPropertyAnnualSubmission.DeleteHistoricUkPropertyAnnualSubmissionRequest
 
 import scala.concurrent.Future
 
-class DeleteHistoricUkPropertyAnnualSubmissionConnectorSpec extends ConnectorSpec {
+class DeleteHistoricUkPropertyAnnualSubmissionConnectorSpec extends ConnectorSpec with MockFeatureSwitches {
 
   private val nino: String       = "AA123456A"
   private val mtdTaxYear: String = "2021-22"
   private val taxYear: TaxYear   = TaxYear.fromMtd(mtdTaxYear)
 
   "connector" must {
-    "send a request and return no content for FHL" in new IfsTest with Test {
-      lazy val propertyType: HistoricPropertyType = HistoricPropertyType.Fhl
+    "send a request and return no content" when {
+      "using FHL data" in new IfsTest with Test {
+        lazy val propertyType: HistoricPropertyType = HistoricPropertyType.Fhl
 
-      val outcome: Right[Nothing, ResponseWrapper[Unit]] = Right(ResponseWrapper(correlationId, ()))
+        MockFeatureSwitches.isPassDeleteIntentEnabled.returns(true)
```

Comment:
For the tests where we have the feature switch enabled, should we override the required headers to include an intent header to make it clear it is getting picked up and passed through?